wqflask/wqflask/correlation/pre_computes.py

```diff
@@ -174,10 +175,10 @@ def fetch_text_file(dataset_name, conn, text_dir=TEXTDIR):
     """fetch textfiles with strain vals if exists"""
 
     with conn.cursor() as cursor:
-        query = 'SELECT Id, FullName FROM ProbeSetFreeze WHERE Name = "%s"' % dataset_name
-        cursor.execute(query)
+        cursor.execute(
+            'SELECT Id, FullName FROM ProbeSetFreeze WHERE Name = "%s"' % dataset_name)
```

Comment:
Change `"%s"` to `%s` (no double quotes) and don't do string interpolation.

Change the entire thing to:

```python
cursor.execute(
    'SELECT Id, FullName FROM ProbeSetFreeze WHERE Name = %s', dataset_name)
```

**NOTE**: the text of the query (`"SELECT  Id, ..."`) is separated from the parameter (`dataset_name`) with a comma (**,**) not a string interpolation operator (**%**)